hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/json/MemberTrigger.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.alerts.api.json;
+
+import java.util.Map;
+
+/**
+ * A convenience class used in the REST API to POST a Member Trigger.
+ *
+ * @author jay shaughnessy
+ * @author lucas ponce
+ */
+public class MemberTrigger {
+
+    private String groupId;
+    private String memberId;
+    private String memberName;
+    private Map<String, String> memberContext;
+    private Map<String, String> dataIdMap;
```

Comment:
I guess MemberTrigger.dataIdMap should be propagated at some point in MemberCondition.dataIdMemberMap, right ?
I imagine that there is an example in the rest of the code that I can study, but the different of types with the inner types makes me wonder if I'm understanding correctly the tokens/substitution algorithm.
Basically I understand that the group has a list of tokens and a list of members that need to be substituted at some point.
The inner map in the MemberCondition.dataIdMemberMap should be also in the upper MemberTrigger.dataIdMap ?